CharacterCreation/03_Feats.txt

```diff
@@ -507,7 +549,60 @@ result in extra hit points, or take the average of the result (1d10 = 5,
 2d10 = 10, can be taken twice). If any of your die rolls is a 1 or 2, reroll
 the die.
 
+== Unarmed Combat Training ==
+
+Prerequisites:
+* STR 5, DEX 5
+* Levle 5, OR Assassin Class, OR Berserker Class
+
+	Increase base and minimum damage of Unarmed Strike by 20. 
+	
+	Additionally, Once per round after a successful melee strike, 
+if the target is not wielding a CQB weapon, you may use a reaction to 
+make a disarm attempt.
+
+== Unarmed Combat Expertise ==
+
+Prerequisites:
+* Unarmed Combat Training
+* STR 6, DEX 6
+* Level 10, OR Assassin Class and Level 5, OR Berserker Class and level 5
+
+	Increase base and minimum damage of Unarmed Strike by 20.
+	
+	Disarm attempts made with Unarmed Combat Training may be made 
+against CQB guns and Two-Handed melee weapons.
+
+== Unarmed Combat Mastery ==
+
+Prerequisites:
+* Unarmed Combat Expertise
+* STR 7, DEX 7, 
+* Level 15, OR Assassin Class and Level 10, OR Berserker Class and Level 10
+
+	Increase base and minimum damage of Unarmed Strike by 20.
+	
+	Whenever you would make a disarm attempt with Unarmed Combat 
+Training, you may instead make a Trip, Shove, or Grapple attempt.
+
+== Unarmed Combat Grand Mastery ==
+
+Prerequisites:
+* Unarmed Combat Mastery
+* STR 7, DEX 7

```

Comment:
What do these comas mean? Do they mean 'and' or 'or'?

Reply:
Should we have 'and' requirements each on their own bullet point?

Reply:
And, will spell out.
Individual bullets are assumed to be ANDed with each other, but I thought it was a sane space-saving mechanism to list all attribute requirements on the same line.